syntax = "proto2";
package actions.proto;

enum AtomicAction {
  AA_UNKNOWN = 0;
  AA_MOVE = 1;
  AA_SELL = 2;
  AA_BUY = 3;
  AA_SWITCH_STATE = 4;
}

message Step {
  optional AtomicAction action = 1;
  optional uint64 connection_id = 2;
  optional string good = 3;
}

message Plan {
  repeated Step steps = 1;
}
